google-cloud-spanner/clirr-ignored-differences.xml

```diff
@@ -192,6 +192,46 @@
     <className>com/google/cloud/spanner/StructReader</className>
     <method>java.util.List getPgJsonbList(java.lang.String)</method>
   </difference>
+  <difference>
```

Comment:
I think these clirr changes are no longer needed, since they were made non-mandatory a few months back. Any particular reason these are failing?

Reply:
As far as I can tell, the check is no longer required, but it is still active. Which means that if you don't add this, the build of the PR will be marked red (and some of us have brains that are not able to handle that, and need a green checkmark to be able to sleep at night ;-) )